src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -0,0 +1,229 @@
+                    ToolbarItems.Add(_deleteItem);
+                }
+                _vm.EditorMargins = new Thickness(0,5,0,0);
+            }
+            if (Device.RuntimePlatform == Device.iOS)
+            {
+                ToolbarItems.Add(_closeItem);
+                if (_vm.EditMode)
+                {
+                    ToolbarItems.Add(_moreItem);
+                }
+                _vm.ShowEditorSeparators = true;
+                _vm.EditorMargins = new Thickness(0,10,0,5);
+                _typePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+                _deletionDateTypePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+                _expirationDateTypePicker.On<iOS>().SetUpdateMode(UpdateMode.WhenFinished);
+            }
+            
+            _typePicker.ItemDisplayBinding = new Binding("Key");
+            _deletionDateTypePicker.ItemDisplayBinding = new Binding("Key");
+            _expirationDateTypePicker.ItemDisplayBinding = new Binding("Key");
+
+            if (_vm.IsText)
+            {
+                _nameEntry.ReturnType = ReturnType.Next;
+                _nameEntry.ReturnCommand = new Command(() => _textEditor.Focus());
+            }
+        }
+        
+        protected override async void OnAppearing()
+        {
+            base.OnAppearing();
+            _broadcasterService.Subscribe(nameof(SendAddEditPage), (message) =>
+            {
+                if (message.Command == "selectFileResult")
+                {
+                    Device.BeginInvokeOnMainThread(() =>
+                    {
+                        var data = message.Data as Tuple<byte[], string>;
+                        _vm.FileData = data.Item1;
+                        _vm.FileName = data.Item2;
+                    });
+                }
+            });
+            await LoadOnAppearedAsync(_scrollView, true, async () =>
+            {
+                var success = await _vm.LoadAsync();
+                if (!success)
+                {
+                    await Navigation.PopModalAsync();
+                    return;
+                }
+                if (!_vm.EditMode && string.IsNullOrWhiteSpace(_vm.Send?.Name))
+                {
+                    RequestFocus(_nameEntry);
+                }
+            });
+        }
+        
+        protected override void OnDisappearing()
+        {
+            base.OnDisappearing();
+            if (Device.RuntimePlatform != Device.iOS)
+            {
+                _broadcasterService.Unsubscribe(nameof(SendAddEditPage));
+            }
+        }
+
+        private void OnMaxAccessCountTextChanged(object sender, TextChangedEventArgs e)
+        {
+            if (string.IsNullOrEmpty(e.NewTextValue))
+            {
+                _vm.MaxAccessCount = null;
+                _maxAccessCountStepper.Value = 0;
+                return;
+            }
+            // accept only digits
+            if (!int.TryParse(e.NewTextValue, out int value))
```

Comment:
the `out int value` here is not used, use a dismissed value name instead.
```suggestion
            if (!int.TryParse(e.NewTextValue, out int _))
```

Reply:
fixed